app/src/main/java/com/globant/scriptsapadea/ui/activities/ScreenPlayActivity.java

```diff
@@ -4,21 +4,20 @@
 import android.content.Context;
 import android.content.Intent;
 import android.os.Bundle;
+import android.support.v4.app.Fragment;
 import android.view.View;
 
 import com.globant.scriptsapadea.R;
-import com.globant.scriptsapadea.ui.fragments.PictureFragment;
+import com.globant.scriptsapadea.interfaces.OnNavigateToFragmentListener;
 import com.globant.scriptsapadea.ui.fragments.ScreenPlayFragment;
-import com.globant.scriptsapadea.ui.fragments.TakePictureFragment;
 
 import roboguice.inject.ContentView;
 
 @ContentView(R.layout.activity_screen)
-public class ScreenPlayActivity extends BaseActivity implements ScreenPlayFragment.OnScreenplayChangeFragmentListener,PictureFragment.OnSetPictureFragmentImageListener  {
+public class ScreenPlayActivity extends BaseActivity implements OnNavigateToFragmentListener {
```

Comment:
@leomendezg If you reused the same interface in every Fragment what happens with the same implementing Activity method ? How does the system knows witch one belong to each fragment if is the same method definition ? Thar's why must define each interface for each Fragment.